### DISCUSSION TITLE
AdminLTE License

### DISCUSSION THREAD

**Discussion by pakorn33** [ColorlibHQ/AdminLTE#4327] [Q&A] [answered]
Dear Team,

I was wondering if I already bought the premium version. It will include adminlte in a purchase order?

I'm currently working on a project using adminlte html free version but the client wants to remove footer so I'm going to buy premium by purchasing "Adminty HTML Dashboard" not sure if this is correct, please advise.

Thank You in Advance,
Best Regards,

**Comment by REJack** [comment, marked as answer]
1. AdminLTE != Adminty 
2. AdminLTE is a Open Source Panel and not Paid, therefore you can set in the footer what ever you or your customer wants.

I hope that helps you 😄

**Reply by pakorn33** [reply]
Thank you. I'm not sure the link description below doesn't match but i understand that the mit license will be the same jquery.

https://colorlibsupport.com/t/adminlte-3-license/28828/3
https://adminlte.io/docs/3.2/license.html
